commands-v2/ramsete/subsystems/drivetrain.py

```diff
@@ -0,0 +1,118 @@
+from commands2 import SubsystemBase
+
+from wpilib import SpeedControllerGroup, PWMVictorSPX, Encoder
+from wpilib.drive import DifferentialDrive
+from wpilib.interfaces import GenericHID
+
+from wpimath.geometry import Pose2d, Rotation2d
+from wpimath.kinematics import DifferentialDriveOdometry, DifferentialDriveWheelSpeeds
+
+from constants import constant
+
+
+class Drivetrain(SubsystemBase):
+    def __init__(self, controller: GenericHID):
+
+        super().__init__()
+
+        frontLeftMotor, backLeftMotor = (
```

Comment:
Stylistically, it seems like it would be easier to just create the SpeedControllerGroup here instead of doing it in two steps.

Reply:
Agreed. Assigned to variable when debugging segfaults. Fixed, thanks!